CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to the Credit default prediction project
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+You can contribute in many ways. This document outlines how to propose a change to the **credit_default_prediction** project. 
+
+# Types of Contributions
+---
+
+### Minor Changes
+Small typos or grammatical errors in documentation may be edited directly using
+the GitHub web interface, as long as the changes are not made in the _source_ file.
+
+*  YES: you edit a roxygen comment in a `.R` file below `R/`.
```

Comment:
do we have such files ?

Reply:
yeah I think since we are working with python files, you can mention ".py" and docstrings instead?

Reply:
@liannah Thanks for highlighting it , I have updated it now. So, basically the idea here is to allow contributors to make docstring or comments changes directly but to restrict editing the source codes without creating issue

Reply:
@Arushi282 Can you do review now ?

Reply:
Looks good to me, I am approving. We can wait for @Arushi282 's approval as well.

Reply:
Hi @karanpreetkaur looks great, thanks for making the changes, and the contributing file is basically how others who are not in the core group can make changes to the project on github. Will merge this now.